docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Back up and restore Platform - Kubernetes
+headerTitle: Back up and restore Platform - Kubernetes
+linkTitle: Back up and restore Yugabyte Platform - K8s
+description: Use a script file to back up and restore Yugabyte Platform on Kubernetes.
+aliases:
+menu:
+  latest:
+    identifier: back-up-restore-k8s
+    parent: administer-yugabyte-platform
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp" class="nav-link">
+      <i class="fas fa-cloud"></i>
+      Default
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+Yugabyte Platform installations include configuration settings, certificates and keys, and other components required for orchestrating and managing YugabyteDB universes.
+
+You can use the Yugabyte Platform backup script to back up an existing Yugabyte Platform server and restore it, when needed, for disaster recovery or migrating to a new server.
+
+## Back up a Yugabyte Platform server
+
+1. Copy the the Yugabyte Platform backup script (`yb_platform_backup.sh`) from the yugabyte-db repository to your local workstation using the following wget command:
+
+    ```sh
+    wget https://github.com/yugabyte/yugabyte-db/blob/master/managed/devops/bin/yb_platform_backup.sh
+    ```
+
+1. Verify that the machine performing the backup operation can access the Yugabyte Platform kubernetes pod instance:
+
+    ```sh
+    kubectl exec --namespace <kubernetes_namespace> -it <kubernetes_pod> -c yugaware -- cat /opt/yugabyte/yugaware/README.md
+    ```
+
+    <br/>
+
+    The Yugabyte Platform console is used in a highly available mode and orchestrates and manages YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For more details, refer to the [Yugabyte Platform overview](https://docs.yugabyte.com/latest/yugabyte-platform/overview/).
```

Comment:
This didn't render correctly in the preview -- it was supposed to represent the output of running the above command (not a new paragraph). And, after looking at it again, it seems kind of confusing. Maybe we can just delete this line?

Reply:
Removed from all the procedures, and stuck it up top in the introduction...

Reply:
:+1: